src/vector/init.ts

```diff
@@ -112,3 +117,33 @@ export async function loadLanguage() {
         console.error("Unable to set language", e);
     }
 }
+
+export async function loadSkin() {
+    // Ensure the skin is the very first thing to load for the react-sdk. We don't even want to reference
+    // the SDK until we have to in imports.
+    console.log("Loading skin...");
+    // load these async so that its code is not executed immediately and we can catch any exceptions
+    const [sdk, skin] = await Promise.all([
+        import(
+            /* webpackChunkName: "matrix-react-sdk" */
+            /* webpackPreload: true */
+            "matrix-react-sdk"),
+        import(
+            /* webpackChunkName: "riot-web-component-index" */
```

Comment:
Are these corresponding in some way to the split up in the graph? Not sure I get the relation, as the names seem to be different.

Reply:
so, Webpack can take its own cues and liberties to further split down the bundles if they are too large, but it'll base the name on what is given here, yes

Reply:
![image](https://user-images.githubusercontent.com/2403652/78552715-e6287b80-77ff-11ea-9435-6460d713107b.png)